cvFit/src/app/macroCalculator/macro-calc-test.spec.ts

```diff
@@ -0,0 +1,20 @@
+import { FormGroup } from "@angular/forms";
+import { CalcService } from "./calculator.service";
+
+describe('CalcService', () => {
+    
+
+    let service: CalcService;
+    
+
+    beforeEach(() => {
+        service = new CalcService();
+    })
+
+    it('should have no values to start', () => {
+        expect(service.dailyCalories).toBe(null);
+    })
+    it('should return male bmr value when calculateBMR_male is called', () =>{
+        //service.calculateBMR_male(service.)
```

Comment:
Let's focus on unit testing next and see if we can get this service tested. Read through some docs: https://angular.io/guide/testing-services and maybe take a look at some pluralsight videos on the topic and let's return to here for next PR.